### PR TITLE
Use iframe uploader to upload study file in order item form. 

### PR DESCRIPTION
**Pull request by fdlk**
Fixes https://github.com/molgenis/molgenis-lifelines/issues/128
Uses https://github.com/cmlenz/jquery-iframe-transport to upload the file with the study data request.

Also: language change: resetted -> reset

### CODE REVIEW

File: molgenis-omx-protocolviewer/src/main/java/org/molgenis/omx/protocolviewer/ProtocolViewerController.java [hunk 1]
```diff
@@ -247,12 +247,22 @@ public String getOrderDataForm()
...
 	{
 		if (!getEnableOrderAction()) throw new MolgenisDataAccessException("Action not allowed");
-		protocolViewerService.submitStudyDefinitionDraftForCurrentUser(name, file, catalogId.toString());
+		try {
+			protocolViewerService.submitStudyDefinitionDraftForCurrentUser(name, file, catalogId.toString());
+			ModelAndView model = new ModelAndView("order-response");
+			model.addObject("ok", true);
+			model.addObject("message", "Your submission has been received.");
+			return model;
+		} catch (Exception ex){
```
Comment (dennishendriksen) [inline molgenis-omx-protocolviewer/src/main/java/org/molgenis/omx/protocolviewer/ProtocolViewerController.java]: did you mean throw RuntimeException ex?

Comment (fdlk) [inline molgenis-omx-protocolviewer/src/main/java/org/molgenis/omx/protocolviewer/ProtocolViewerController.java]: The IFrame solution isn't able to access the HTTP status codes, and will always call the complete() callback. So the idea was to put the response in a success message instead.

Comment (fdlk) [inline molgenis-omx-protocolviewer/src/main/java/org/molgenis/omx/protocolviewer/ProtocolViewerController.java]: You're absolutely right! thanks!

Comment (fdlk) [inline molgenis-omx-protocolviewer/src/main/java/org/molgenis/omx/protocolviewer/ProtocolViewerController.java]: Will update to make clearer.

**Pull request merged**